umbraco-cloud/troubleshooting/deployments/changes-not-being-applied.md

```diff
@@ -59,7 +59,7 @@ An example of a faulty config transform not being applied could look something l
 
 1. Verify the Config Transform file mentioned in the error message and ensure the XML is valid.
 2. Update the Config Transform file as necessary on your local solution and deploy the change to the Cloud environments.
-3. Run a ['Schema Deployment From Data Files'](../../deployment/deploy-operations/extract-schema-to-data-files.md) to extract the schema that you previously deployed.
+3. Run a **Schema Deployment From Data File** to extract the schema that you previously deployed.
```

Comment:
Would it make sense to link to this: https://docs.umbraco.com/umbraco-cloud/deployments/deploy-dashboard#deploy-operations

Not sure what happened to the article referenced here, but I believe it was linking to a place that explained the deployment process also mentioned in the article I linked above 🤔

Reply:
Yup, makes sense.